src/createAnimatedComponent/createAnimatedComponent.tsx

```diff
@@ -56,12 +56,14 @@ import {
 } from '../reanimated2/layoutReanimation/web';
 import type { CustomConfig } from '../reanimated2/layoutReanimation/web/config';
 import type { FlatList, FlatListProps } from 'react-native';
+import { setObserver } from '../reanimated2/layoutReanimation/web/domUtils';
 
 const IS_WEB = isWeb();
 const IS_FABRIC = isFabric();
 
 if (IS_WEB) {
   configureWebLayoutAnimations();
+  setObserver();
```

Comment:
It is possible to call setObserver() only when someone uses exiting animation?

Reply:
We can, but then we will call it more than once and we will have to check whether observer was already set.